ado_asana_sync/sync/app.py

```diff
@@ -53,17 +55,26 @@ class App:
 
     def __init__(
         self,
-        ado_pat: str = "",
-        ado_url: str = "",
-        asana_token: str = "",
-        asana_workspace_name: str = "",
+        ado_pat: Optional[str] = None,
+        ado_url: Optional[str] = None,
+        asana_token: Optional[str] = None,
+        asana_workspace_name: Optional[str] = None,
+        applicationinsights_connection_string: Optional[str] = None,
     ) -> None:
         self.ado_pat = ado_pat or os.environ.get("ADO_PAT")
         self.ado_url = ado_url or os.environ.get("ADO_URL")
         self.asana_token = asana_token or os.environ.get("ASANA_TOKEN")
         self.asana_workspace_name = asana_workspace_name or os.environ.get(
             "ASANA_WORKSPACE_NAME"
         )
+        self.applicationinsights_connection_string = (
+            applicationinsights_connection_string
+            or os.environ.get("APPLICATIONINSIGHTS_CONNECTION_STRING", None)
+        )
+        if not self.applicationinsights_connection_string:
+            os.environ["OTEL_LOGS_EXPORTER"] = "None"
+            os.environ["OTEL_METRICS_EXPORTER"] = "None"
+            os.environ["OTEL_TRACES_EXPORTER"] = "None"
```

Comment:
**Suggestion:** Instead of setting environment variables inside the `__init__` method, consider using a configuration file or command line arguments. This would make the code more flexible and easier to maintain.
```suggestion
        # Suggestion: Move these settings to a configuration file or command line arguments
```